### PR TITLE
Example: Add example

### PR DESCRIPTION
**Pull request by techmsi**
#24 
#21 
https://trello.com/c/cKNRlqE9/16-add-an-example-generator

### CODE REVIEW

File: generators/example/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {
+          options: {
+            description: 'This is an example description for this package.json.',
+            keywords: 'example, generators'
+          },
+          args: [example.appName]
+        }
+      );
+    },
+    createComponent: function () {
```
Comment (techmsi) [inline generators/example/index.js]: Creates
* template.nunjucks, all.css, print.css

* schema.yml
```
headline:
  _has: text
```
bootstrap.yml
```
components:
  hello-world:
    headline: ''
```

@yoshokatana Anything else needed?

File: generators/example/templates/_index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+## It Works!
```
Comment (techmsi) [inline generators/example/templates/_index.html]: @yoshokatana This file replaces `component/ hello-world/template.nunjucks`, correct?

**Comment by yoshokatana** [comment]
Oh...It shouldn't be a subgenerator. It should just be part of the `clay` app generator.

**Comment by techmsi** [comment]
@yoshokatana Code is cleaner as a sub generator that runs all 3 other generators. It is also clear to the user that this is an example they can demo.

**Comment by techmsi** [comment]
Closing for now since, this depends on us making our open-source layout component.

**Pull request closed without merging**